specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/Dataset.json

```diff
@@ -552,21 +691,22 @@
     },
     "ParquetDatasetTypeProperties": {
       "description": "Parquet dataset properties.",
+      "type": "object",
       "properties": {
         "location": {
           "$ref": "#/definitions/DatasetLocation",
           "description": "The location of the parquet storage."
         },
         "compressionCodec": {
-          "type": "string",
+          "type": "object",
           "enum": [
             "none",
             "gzip",
             "snappy",
             "lzo"
           ],
           "x-ms-enum": {
-            "name": "ParquetCompressionCodec",
+            "name": "compressionCodec",
```

Comment:
I see a few enums with the same names but different values. It feels like if they are different they should have different names. Why is it changing from `ParquetCompressionCodec` to `compressionCodec`

Reply:
This was renamed in ADF swagger so we are just trying to keep both Synapse and ADF in sync

Reply:
I suspect this is bug in ADF Swagger spec because I can still see "avroCompressionCodec", "orcCompressionCodec" is the Swagger spec. It doesn't make much sense to me that we only change the "parquet" one. I guess in the original ADF change, the developer wanted to change it to be "parquetCompressionCodec". @drosales57 , could you please check this with someone in ADF team?